packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -20,6 +20,24 @@ import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.da
 import 'package:shorebird_cli/src/version.dart';
 import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
 
+/// {@template patch_artifact_fingerprint}
+///
+/// Patch artifact fingerprint that we are about to upload.
+///
+/// {@endtemplate}
+class PatchArtifactFingerprint {
+  const PatchArtifactFingerprint({
```

Comment:
I think we're going to eventually want to just embed this in the patch, rather than storing it separately on the server.  This might be the right thing for now though.